clients/js-legacy/test/ciphertextCiphertextEquality.ts

```diff
@@ -39,18 +40,57 @@ describe('ciphertextCiphertextEquality', () => {
 
     it('verify proof data', async () => {
         const transaction = new Transaction().add(
-            createVerifyCiphertextCiphertextEqualityInstruction(
-                testFirstElGamalKeypair,
-                testSecondElGamalPubkey,
-                testFirstElGamalCiphertext,
-                testSecondElGamalCiphertext,
-                testSecondPedersenOpening,
-                testAmount,
-            ),
+            createVerifyCiphertextCiphertextEqualityInstruction({
+                firstKeypair: testFirstElGamalKeypair,
+                secondPubkey: testSecondElGamalPubkey,
+                firstCiphertext: testFirstElGamalCiphertext,
+                secondCiphertext: testSecondElGamalCiphertext,
+                secondOpening: testSecondPedersenOpening,
+                amount: testAmount,
+            }),
         );
         await sendAndConfirmTransaction(connection, transaction, [payer]);
     });
 
+    it('read proof data record', async () => {
+        const recordAccount = Keypair.generate();
+        const recordAccountAddress = recordAccount.publicKey;
+        const recordAuthority = Keypair.generate();
+
+        const proofData = CiphertextCiphertextEqualityProofData.new(
+            testFirstElGamalKeypair,
+            testSecondElGamalPubkey,
+            testFirstElGamalCiphertext,
+            testSecondElGamalCiphertext,
+            testSecondPedersenOpening,
+            testAmount,
+        );
+
+        await createInitializeWriteRecord(
+            connection,
+            payer,
+            recordAccount,
+            recordAuthority,
+            BigInt(0),
+            proofData.toBytes(),
+        );
+
+        const recordAccountInfo: RecordAccountInfo = {
+            account: recordAccountAddress,
+            offset: RECORD_META_DATA_SIZE,
+        };
+
+        const transaction = new Transaction().add(
+            createVerifyCiphertextCiphertextEqualityInstruction(recordAccountInfo),
+        );
+        await sendAndConfirmTransaction(connection, transaction, [payer]);
+
+        const destinationAccount = Keypair.generate();
+        const destinationAccountAddress = destinationAccount.publicKey;
+
+        await closeRecord(connection, payer, recordAccountAddress, recordAuthority, destinationAccountAddress);
```

Comment:
Up to you, but if you're going to close the account in the test, may as well check that it got closed, similar to the context state accounts in the other tests

Reply:
Makes sense. Done!